src/control/drpc/drpc_server.go

```diff
@@ -136,7 +136,10 @@ func (d *DomainSocketServer) Shutdown() {
 // RegisterRPCModule takes a Module and associates it with the given
 // DomainSocketServer so it can be used to process incoming dRPC calls.
 func (d *DomainSocketServer) RegisterRPCModule(mod Module) {
-	d.service.RegisterModule(mod)
+	if err := d.service.RegisterModule(mod); err != nil {
```

Comment:
If this can't happy should the prototype not return an error instead?

Reply:
Good question, seems like maybe having RegisterModule panic would probably be better, because I think the only failure would be trying to register the same module twice, and that seems like a programming error rather than something that can be handled.